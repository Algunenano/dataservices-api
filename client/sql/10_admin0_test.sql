-- Mock the server function
CREATE OR REPLACE FUNCTION cdb_geocoder_server.geocode_admin0_polygons(user_id name, tx_id bigint, country_name text)
RETURNS Geometry AS $$
BEGIN
  RAISE NOTICE 'cbd_geocoder_server.geocode_admin0_polygons invoked with params (%, %, %)', user_id, 'some_transaction_id', country_name;
  RETURN NULL;
END;
$$ LANGUAGE 'plpgsql';


-- Exercise the public and the proxied function
SELECT cdb_geocoder_client.geocode_admin0_polygons('Spain');
