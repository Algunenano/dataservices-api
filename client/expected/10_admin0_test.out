-- Add to the search path the schema
SET search_path TO public,cartodb,cdb_geocoder_client;
-- Mock the server function
CREATE OR REPLACE FUNCTION cdb_geocoder_server.cdb_geocode_admin0_polygon(username text, country_name text)
RETURNS Geometry AS $$
BEGIN
  RAISE NOTICE 'cdb_geocoder_server.cdb_geocode_admin0_polygon invoked with params (%, %)', username, country_name;
  RETURN NULL;
END;
$$ LANGUAGE 'plpgsql';
-- Exercise the public and the proxied function
SELECT cdb_geocode_admin0_polygon('Spain');
NOTICE:  cdb_geocoder_client._cdb_geocode_admin0_polygon(2): [contrib_regression] REMOTE NOTICE: cdb_geocoder_server.cdb_geocode_admin0_polygon invoked with params ("test_user", Spain)
CONTEXT:  SQL statement "SELECT cdb_geocoder_client._cdb_geocode_admin0_polygon(username, country_name)"
PL/pgSQL function cdb_geocode_admin0_polygon(text) line 13 at SQL statement
 cdb_geocode_admin0_polygon 
----------------------------
 
(1 row)

